addon/content/js/jquery.tokeninput.js

```diff
@@ -259,7 +259,7 @@ $.TokenList = function (input, settings) {
 
   // Pre-populate list if items exist
   function init_list () {
-    li_data = settings.prePopulate;
+    var li_data = settings.prePopulate;
```

Comment:
Similar to handlebars, this is also an imported vendor file that's going away soon. Lets just drop this change please.

Reply:
Thanks, dropped